starboard/linux/x64x11/main.cc

```diff
@@ -31,6 +32,9 @@
 #include "third_party/crashpad/wrapper/wrapper.h"
 
 extern "C" SB_EXPORT_PLATFORM int main(int argc, char** argv) {
+  // Set M_ARENA_MAX to a low value to slow memory growth due to fragmentation.
+  mallopt(M_ARENA_MAX, 2);
```

Comment:
Not sure if you want to fail on return like raspi: https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:starboard/raspi/shared/main.cc;l=35?q=p:cobalt%20M_ARENA_MAX,%202

Reply:
Tried that, and found some unexpected behavior. It does set the arenas but doesn't always yield correct return codes, so i removed the DCHECK as it's more of a hint anyway.